workers/unity/Assets/Gdk/Core/MutableView/MutableView.cs

```diff
@@ -211,8 +233,12 @@ public void AddGameObjectEntity(Entity entity, GameObject gameObject)
                 case Authority.Authoritative:
                     if (!HasComponent<NotAuthoritative<T>>(entity))
                     {
-                        Debug.LogErrorFormat(Errors.IncorrectAuthorityTransition, Authority.Authoritative,
-                            Authority.NotAuthoritative);
+                        ErrorDispatcher.HandleError(new InvalidStateTransition
+                        {
+                            NewAuthority = Authority.Authoritative,
+                            OldAuthority = Authority.NotAuthoritative,
+                            Attribute = "authority"
```

Comment:
as I don't see the actual implementation, it's a bit hard to tell whether `Attribute` is really needed. If yes, don't use a string, but create an enum

Reply:
Changed to be more generic.